finout/CODEOWNERS

```diff
@@ -0,0 +1,2 @@
+# This repo is just for the tile, code owner of the integration may be different
```

Comment:
You can delete this file.

The proper place for CODEOWNERS is at `.github/CODEOWNERS`.  See this PR for an example: https://github.com/DataDog/integrations-extras/pull/1218/files#diff-3d36a1bf06148bc6ba1ce2ed3d19de32ea708d955fed212c0d27c536f0bd4da7

Reply:
Hi @mgarabed, done

Reply:
Build failing again tough, as the CODEOWNERS is valid but again my user have no access to this repository

Reply:
Hi @roiravhon you'll want to edit the existing file here: [CODEOWNERS](https://github.com/DataDog/integrations-extras/blob/master/.github/CODEOWNERS)

And add a line like this:

```
/finout/ @roiravhon some-email-address@goes-here.com
```

Then delete the file you have at `finout/.github/CODEOWNERS`.

Reply:
@mgarabed I was slow on understanding this one :)
Sorry about that - fixed now, hope its ok

Reply:
Looks good, thanks!